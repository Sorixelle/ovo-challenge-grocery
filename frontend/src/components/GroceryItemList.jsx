import React from 'react'

const GroceryItemList = ({ children }) => {
  return (
    <div className='list-group'>
      {children}
    </div>
  )
}

export default GroceryItemList
