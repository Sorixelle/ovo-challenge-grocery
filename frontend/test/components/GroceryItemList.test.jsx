import React from 'react'
import { render } from '@testing-library/react'
import GroceryItemList from '../../src/components/GroceryItemList'

describe('GroceryItemList', () => {
  const node = (
    <GroceryItemList>
      <p>Test</p>
    </GroceryItemList>
  )

  it('should match the snapshot', () => {
    const { container } = render(node)
    expect(container).toMatchSnapshot()
  })
})
