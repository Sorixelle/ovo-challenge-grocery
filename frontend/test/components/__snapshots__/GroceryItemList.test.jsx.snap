// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroceryItemList should match the snapshot 1`] = `
<div>
  <div
    class="list-group"
  >
    <p>
      Test
    </p>
  </div>
</div>
`;
